sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/implementation/VirtualMachineImpl.java

```diff
@@ -2852,6 +2936,46 @@ void setDefaultEncryptionSet(String diskEncryptionSetId) {
             this.defaultDiskEncryptionSet = new DiskEncryptionSetParameters().withId(diskEncryptionSetId);
         }
 
+        void setDeleteOptions(DeleteOptions deleteOptions) {
```

Comment:
private? Does it get called?

Reply:
Thanks, they got added in 3-1 implementation. Will remove.